### PR TITLE
Add conditional to create cloud-init.yaml.

### PR DESCRIPTION
**Pull request by jasondoze**
Add noninteractive to brew install.

Add multipass driver set to Qemu.

Add cloud-init to relativepath launch.

Add SSH into relativepath VM.

Add .gitignore.

### CODE REVIEW

**Review by ChrisForti — APPROVED** [review]
This looks good Jason more simple. than what I had envisioned. So does this script run automatically? or is it something we do manually, by command? In reference to the gitignore.

**Comment by jasondoze** [comment]
@ChrisForti 
Run the script the same as before in your terminal with `bash darwin.sh`

File: darwin.sh [hunk 1]
```diff
@@ -1,61 +1,86 @@
...
-  ssh-keygen -t ed25519 -f "./id_ed25519" -N ''
+  echo -e "\n==== Creating cloud-init.yaml ====\n"
+  cat <<- EOF > cloud-init.yaml
+users:
+  - default
+  - name: $USER
+    sudo: ALL=(ALL) NOPASSWD:ALL
+    shell: /bin/bash
+    ssh_authorized_keys:
+      - $(cat id_ed25519.pub)
```
Comment (ChrisForti) [inline darwin.sh:70]: I was trying to put the actual key in here... this looks good

File: darwin.sh [hunk 2]
```diff
@@ -1,61 +1,86 @@
...
+  echo -e "\n==== Relativepath VM present ====\n"
+else 
+  echo -e "\n==== Creating Relativepath VM ====\n"
+  multipass launch --name relativepath --cloud-init cloud-init.yaml
 fi
 
-echo "launching multipass vm"
-multipass launch --name relativepath
+# SSH into Relativepath VM
+ssh -o StrictHostKeyChecking=no -i "./id_ed25519" $USER@$(multipass info relativepath | grep IPv4 | awk '{ print $2 }')
```
Comment (ChrisForti) [inline darwin.sh:84]: This must be the alternative to the IP address. sweet!

**Comment by ChrisForti** [comment]
usr/local/bin/multipass
multipass already installed
creating ssh key pair
Generating public/private ed25519 key pair.
Your identification has been saved in ./id_ed25519
Your public key has been saved in ./id_ed25519.pub
The key fingerprint is:
SHA256:FLVwH0wL0VgJHErEwZ82K5twLAHn72YXSIfPtxQQiRk newuser@News-MBP.lan
The key's randomart image is:
+--[ED25519 256]--+
|       +EBO%+.   |
|    . ..===++o   |
|     +  oo.oo    |
|      o.o * .    |
|       =S* o .   |
|      o * = o    |
|       = + + .   |
|        * . .    |
|       o .       |
+----[SHA256]-----+
launching multipass vm
Launched: relativepath                                                          

Works like a charm
@Jason-Doze

File: darwin.sh (hunk 2 above)
Comment (jasondoze) [reply darwin.sh:84]: This IS the IP address, here we are grepping the IP (IP version 4) from the output of multipass info relativepath and capturing the IP address in a subshell. Pretty dope!

Comment (ChrisForti) [reply darwin.sh:84]: Yes it is!

**Pull request merged**